cpu/kinetis_common/adc.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @addtogroup  cpu_kinetis_common Common Drivers for Freescale Kinetis MCUs
```

Comment:
Change to `@ingroup cpu_kinetis_common`, put the textual description of the group in the `@defgroup` command, create a doc.txt similar to what some other CPU directories have.

This applies for all other uses of `@addtogroup cpu_kinetis_common` below as well.